lib/chef/data_bag_item.rb

```diff
@@ -107,9 +107,10 @@ def self.object_name(data_bag_name, id)
     end
 
     def to_hash
-      result = self.raw_data
+      result = self.raw_data.dup
+      #result = self.raw_data
       result["chef_type"] = "data_bag_item"
-      result["data_bag"] = self.data_bag
+      result["data_bag"] = self.data_bag.to_s
```

Comment:
I had to change this to a string, when I added the `.dup` to self.raw_data on line 110, the data bag key became a symbol instead of a string and caused a unit test to fail.